.circleci/config.yml

```diff
@@ -2,6 +2,7 @@ version: 2
 jobs:
   build:
     docker:
+      # TODO: use image that has softhsm installed. can just apt-get install libsofthsm2 on circleci/golang:1.12
```

Comment:
https://github.com/choria-io/packager add it to one of those and we can switch to those when building

Reply:
Done! It works!